docs/usage-notes.md

```diff
@@ -54,6 +55,30 @@ There are 2 methods to achieve this:
   - Note: the GUI menus may differ slightly on different Linux distributions.
 - From the command line: Use `chmod +x CATcher-x.y.z.AppImage`
 
+
+# Customising the available sessions 
+
+If your sessions are not present in the default dropdown list on CATcher's startup page, you can load custom sessions by clicking on the **file icon** beside the session dropdown. 
+
+Following which, submit a json file with the name `profiles.json`, where the format is specified below.
+
+```json
+{
+    "profiles": [
+        {
+            "profileName": "CATcher", 
+            "encodedText": "CATcher-org/public_data"
+        }
+    ]
+}
+```
+
+The json supplied should only consist of **one key-pair value**, where the key is `"profiles"` and the value is an array of `Profiles` supplied, where each `Profile` is an object containing the `profileName` and `encodedText` fields. 
```

Comment:
```suggestion
The json supplied should only consist of **one key-pair value**, where the key is `"profiles"` and the value is an array of `Profiles`, where each `Profile` is an object containing the `profileName` and `encodedText` fields. 
```

I think the supplied feels abit off here?

Reply:
👍